src/regression.jl

```diff
@@ -88,7 +89,7 @@ function fitRegression(df, lossFunction::Function = proportion_loss; wL0f = fals
     if !Optim.converged(fit)
         @warn "Fitting did not converge"
     end
-    return fit
+    return p_init
```

Comment:
This is the ugly part: I kept the optimization function but right now I'm still using the initial value as output, and this will give us the `_nnls.pdf` you have seen. The optimizer will take a while to be fixed. Do we want to merge this for now?